test/acceptance/tests.py

```diff
@@ -75,6 +75,9 @@ class OpenAssessmentTest(WebAppTest):
         'full_workflow_staff_required':
             u'courses/{test_course_id}/courseware/'
             u'8d9584d242b44343bc270ea5ef04ab03/0b0dcc728abe45138c650732af178afb/'.format(test_course_id=TEST_COURSE_ID),
+        'multiple_ora':
+            u'courses/{test_course_id}/courseware/'
+            u'8d9584d242b44343bc270ea5ef04ab03/0b0dcc728abe45138c650732af178afb'.format(test_course_id=TEST_COURSE_ID),
```

Comment:
@efischer19 Yes you are right. I was having difficulties reproducing the the problems structure as it is currently in test course which seems outdated see [TNL-4371](https://openedx.atlassian.net/browse/TNL-4371)

But I got past through that somehow. I have added the new unit page containing multiple ora problems.